realm/realm-library/src/main/java/io/realm/internal/Property.java

```diff
@@ -29,23 +29,92 @@
     public static final boolean REQUIRED = true;
     public static final boolean INDEXED = true;
 
+    @SuppressWarnings("WeakerAccess")
```

Comment:
public because of the static assertion check here https://github.com/realm/realm-java/pull/5058/files/857c93b34938a1553fc304e4979fa6662618600d#diff-aafc664a99eb76a4271b81acd4f4a44cR27